### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by jamesianberry**
Mostly just proof-reading corrections and formatting changes

### CODE REVIEW

File: documentation/Documentation.md [hunk 1]
```diff
@@ -25,12 +24,12 @@ Documentation:
 * [Technical Specifications for the PMS integration](FHIRDocumentation.md)
 
 
-## Patient Focused Documentation
+## Patient-Focused Documentation
```
Comment (vadi2) [inline documentation/Documentation.md]:
```suggestion
## Patient-focused Documentation

```

Perhaps?

File: documentation/FHIRDocumentation.md [hunk 2]
```diff
@@ -1,13 +1,10 @@
...
-Practice Management System (PMS).
+The Application uses a FHIR R4 interface to communicate with the local Practice Management System (PMS).
 
-In a GP practice setting with a compatible PMS (i.e. Medical Director or Best Practice),
-the application will stand up its own FHIR server in order to communicate with the PMS.
+In a GP practice setting with a compatible PMS (i.e. Medical Director or Best Practice), the application will stand up its own FHIR server in order to communicate with the PMS.
 
-Alternatively, a different FHIR Server may be specified in the configuration. 
-Any FHIR Server must conform to these expectations documented here.
+Alternatively, a different FHIR Server may be specified in the configuration. Any FHIR Server must conform to the expectations documented here.
```
Comment (vadi2) [inline documentation/FHIRDocumentation.md]:
```suggestion
Alternatively, a different FHIR Server may be specified in the configuration. Any FHIR Server used in the end must conform to the expectations documented here.

```

Perhaps this is a clearer message.

**Review by vadi2 — APPROVED** [review]
This looks excellent.

File: documentation/Documentation.md (hunk 1 above)
Comment (jamesianberry) [reply documentation/Documentation.md]: Agreed. Note: http://www.future-perfect.co.uk/grammar-tip/is-it-focussed-or-focused/

**Pull request merged**